### PR TITLE
deletes slavery

### PR DESCRIPTION
**Pull request by TMTIME**
:cl: TMTIME
fix: People can no longer be sold, along with other blacklisted items
/:cl:

### CODE REVIEW

**Comment by Cdey78** [comment]
Blasphemy

**Comment by AuroranAI** [comment]
Emergent gameplay tbh

**Comment by Qustinnus** [comment]
ftl has slavery tho :^)

**Comment by Qustinnus** [comment]
add a check for species so you can sell lizard and catpeople players

File: code/modules/cargo/exports.dm [hunk 1]
```diff
@@ -38,6 +38,8 @@ Credit dupes that require a lot of manual work shouldn't be removed, unless they
 	// We go backwards, so it'll be innermost objects sold first
 	for(var/i in reverseRange(contents))
```
Comment (Qustinnus) [inline code/modules/cargo/exports.dm]: fuck i didnt pay attention

**Pull request merged**